packages/paste-core/components/sidebar/stories/full-composition/console.stories.tsx

```diff
@@ -0,0 +1,214 @@
+import {ProductMessagingIcon} from '@twilio-paste/icons/esm/ProductMessagingIcon';
+import {ProductVideoIcon} from '@twilio-paste/icons/esm/ProductVideoIcon';
+import {ProductLookupIcon} from '@twilio-paste/icons/esm/ProductLookupIcon';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {MoreIcon} from '@twilio-paste/icons/esm/MoreIcon';
+// ONLY for storybook stacked view not to complain on duplicates. aria-label should be carefully selected strings
+import {useUID} from '@twilio-paste/uid-library';
+import {Topbar, TopbarActions} from '@twilio-paste/topbar';
+import {Menu, MenuButton, MenuItem, useMenuState} from '@twilio-paste/menu';
+
+import {
+  Sidebar,
+  SidebarBody,
+  SidebarHeader,
+  SidebarHeaderLabel,
+  SidebarHeaderIconButton,
+  SidebarCollapseButton,
+  SidebarFooter,
+  SidebarPushContentWrapper,
+  SidebarNavigation,
+  SidebarNavigationItem,
+  SidebarNavigationDisclosure,
+  SidebarNavigationDisclosureHeading,
+  SidebarNavigationDisclosureHeadingWrapper,
+  SidebarNavigationDisclosureContent,
+  SidebarNavigationSeparator,
+  SidebarBetaBadge,
+} from '../../src';
+import {AccountSwitcherMenu} from './components/AccountSwitcher';
+import {TrialBadge} from './components/TrialBadge';
+import {SearchBox} from './components/SearchBox';
+import {SupportMenu} from './components/SupportMenu';
+import {AppSwitcher} from './components/AppSwitcher';
+import {UserDialogExample} from './components/UserDialog';
+import {AccountMenu} from './components/AccountMenu';
+import {BillingMenu} from './components/BillingMenu';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Sidebar/FullCompositions',
+};
+
+const onClick = (): void => {};
+
+/* eslint-disable react/jsx-max-depth */
+export const Console: StoryFn = () => {
+  const id = useUID();
+  const menu = useMenuState();
+  const menuLookup = useMenuState();
+  const menuVideo = useMenuState();
+
+  const [pushSidebarCollapsed, setPushSidebarCollapsed] = React.useState(false);
+
+  return (
+    <Box>
+      {/* Can be placed anywhere - position fixed */}
+      <Sidebar aria-label={id} collapsed={pushSidebarCollapsed} variant="compact">
+        <SidebarHeader>
+          <SidebarHeaderIconButton as="a" href="#">
+            <LogoTwilioIcon size="sizeIcon20" decorative={false} title="Go to Console homepage" />
+          </SidebarHeaderIconButton>
+          <SidebarHeaderLabel>Twilio Console</SidebarHeaderLabel>
+        </SidebarHeader>
+        <SidebarBody>
+          <SidebarNavigation hierarchical hideItemsOnCollapse>
+            <SidebarNavigationDisclosure>
+              <SidebarNavigationDisclosureHeadingWrapper>
+                <SidebarNavigationDisclosureHeading icon={<ProductMessagingIcon decorative />} selected>
+                  Messaging
+                </SidebarNavigationDisclosureHeading>
+                <MenuButton {...menu} variant="inverse_link" size="icon_small">
```

Comment:
chore: Menu buttons should be size="reset" to match line height of text and beta badge height

Reply:
ignore